backend/cfg/vectorize.ml

```diff
@@ -255,8 +1331,28 @@ let cfg ppf_dump cl =
   if !Flambda_backend_flags.dump_vectorize
   then Format.fprintf ppf_dump "*** Vectorization@.";
   let cfg = Cfg_with_layout.cfg cl in
-  let dependency_graph = Dependency_graph.from_cfg cfg in
-  if !Flambda_backend_flags.dump_vectorize
-  then Dependency_graph.dump ppf_dump dependency_graph cl;
+  let layout = Cfg_with_layout.layout cl in
+  let cfg_with_infos = Cfg_with_infos.make cl in
```

Comment:
(It is fine for now, but maybe leave a
CR-someday that the function could
or should take a `Cfg_with_infos.t`.)

Reply:
done